LICENSE.txt

```diff
@@ -187,7 +187,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright [yyyy] [name of copyright owner]
+   Copyright 2018 Expedia Inc.
```

Comment:
This file is supposed to be an exact copy of https://www.apache.org/licenses/LICENSE-2.0.txt and shouldn't be changed in any way so please put this back.

Reply:
done